src/frontend/components/Header/Header.css

```diff
@@ -77,15 +76,15 @@
 }
 .responsive__nav {
   height: 100vh;
-  width: 100vw;
+  width:90vw;
   z-index: 999;
```

Comment:
You are using 999as index value, It might cause issues if you try to add z-index in some other component. It will be hard to maintain.

Reply:
okay will correct it.Thanks